elvis.config

```diff
@@ -0,0 +1,22 @@
+[
+ {
+   elvis,
+   [
+    {config,
+     [#{dirs => ["src"],
+        filter => "hocon.erl",
```

Comment:
change to `"*.erl"`
and add  `ignore => [hocon_parser, hocon_scanner]` to ignore them.

Reply:
```
      #{dirs => ["src"],
        filter => "*.erl",
        ruleset => erl_files,
        ignore => [hocon_parser, hocon_scanner]
      },
```

Reply:
ty for the hint. fixed